src/smart-components/Thread/components/ThreadMessageInput/index.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../../../styles/variables';
+
+.sendbird-thread-message-input .sendbird-mention-suggest-list {
+  width: 100%;
+  margin-left: 0px;
+  margin-right: 0px;
+}
+.sendbird-thread-message-input .sendbird-mention-suggest-list .sendbird-mention-suggest-list__user-item {
+  padding-left: 16px;
+  padding-right: 16px;
+}
+.sendbird-thread-message-input .sendbird-mention-suggest-list .sendbird-mention-suggest-list__user-item .sendbird-mention-suggest-list__user-item__nickname {
+  max-width: 134px;
+}
+.sendbird-thread-message-input .sendbird-mention-suggest-list .sendbird-mention-suggest-list__user-item .sendbird-mention-suggest-list__user-item__user-id {
```

Comment:
nit/later - simplify these styles, maybe we need to start with sendbird-mention-suggest-list